testcases/OpTestLPM.py

```diff
@@ -44,11 +46,45 @@ def setUp(self):
         conf = OpTestConfiguration.conf
         self.cv_SYSTEM = conf.system()
         self.console = self.cv_SYSTEM.console
+        self.cv_HOST = conf.host()
         self.cv_HMC = self.cv_SYSTEM.hmc
         self.src_mg_sys = self.cv_HMC.mg_system
         self.dest_mg_sys = self.cv_HMC.tgt_mg_system
+        self.oslevel = None
+
+    def check_pkg_installation(self):
+        pkg_found = True
+        pkg_notfound= []
+        self.oslevel = self.cv_HOST.host_get_OS_Level()
+        lpm_pkg_list = ["src", "rsct.core", "rsct.core.utils", "rsct.basic", "rsct.opt.storagerm", "DynamicRM"]
+        for pkg in lpm_pkg_list:
+            pkg_status = self.cv_HOST.host_check_pkg_installed(self.oslevel, pkg)
+            if not pkg_status:
+                pkg_found = False
+                pkg_notfound.append(pkg)
+        if pkg_found:
+            return True
+        raise OpTestError("Install the required packages : %s" % pkg_notfound)
+
+    def lpm_setup(self):
+        try:
+            self.cv_HOST.host_run_command("systemctl status firewalld.service")
+            self.firewall_status = True
+        except CommandFailed as cf:
+            if cf.exitcode == 3:
+                self.firewall_status = False
+        if self.firewall_status:
+             self.cv_HOST.host_run_command("systemctl stop firewalld.service")
```

Comment:
The command works for redhat, but for suse 
` # SuSEfirewall2  stop`
`SuSEfirewall2[10347]: Firewall rules unloaded.`

Reply:
@harihare you may have this in a separate commit.. but note it in your TODO